### DISCUSSION TITLE
SAML 2.0 connection ACS & URL Entity ID

### DISCUSSION THREAD

**Discussion by iftachbyaf** [requarks/wiki#4926] [Help / Questions]
Hey,

Installed Wikijs deployment and trying to enable SAML2.0 Authentication.
I need to provide on my SAML 2.0 provider the ACS & URL Entity ID but I cannot find them on wikijs. Has anyone run into this and can help me out? :)

Thanks

**Comment by rafaelmathieu** [comment]
did you ever find the ACS url?
